docs/SMCWG-charter.md

```diff
@@ -0,0 +1,328 @@
+announcement to include the basis upon which the suspension has been made.
+
+A suspended Member who believes it has re-met the Eligibility to Participate
+criteria as outlined in this charter shall post its evidence to the SMCWG Public
+Mail List or provide evidence to the SMCWG Chair who SHALL post it to the SMCWG
+Public Mail List. The SMCWG Chair will examine the evidence and unsuspend the
+member, or not, by announcement to the SMCWG Public Mail List. A Member’s
+membership will automatically cease six months after the SMCWG Chair’s
+announcement of its suspension if the Member has not re-met the membership
+criteria by that time.
+
+While suspended, a Member may participate in SMCWG Meetings, SMCWG
+Teleconferences, and on the SMCWG’s discussion lists, but may not propose or
+endorse ballots or take part in any form of voting.
+
+Votes cast before the announcement of a Member’s suspension will stand.
+
+### 5. Membership Declaration
+
+In accordance with the IPR Policy, Members that choose to participate in the
+SMCWG MUST declare their participation and SHALL do so prior to participating.
+The Chair of the SMCWG SHALL establish a list for declarations of participation
+and manage it in accordance with the Bylaws, the IPR Policy, and the IPR
+Agreement.
+
+### 6. Voting and Other Organizational Matters
+
+#### 6.1. Voting Structure
+
+The rules described in Bylaw 2.3 and 2.4 SHALL apply to all ballots, including
+Draft Guideline Ballots.
+
+In order for a ballot to be adopted by the SMCWG, two-thirds or more of the
+votes cast by the Certificate Issuers must be in favor of the ballot and more
+than 50% of the votes cast by the Certificate Consumers must be in favor of the
+ballot. At least one member of each class must vote in favor of a ballot for it
+to be adopted. Quorum is the average number of Member organizations (cumulative,
+regardless of Class) that have participated in the previous three (3) SMCWG
+Meetings or Teleconferences (not counting subcommittee meetings thereof). For
+transition purposes, if three (3) meetings have not yet occurred, quorum is
+three (3).
+
+#### 6.2. Other Organizational Matters
+
+- The Chair MAY delegate any of their duties to the Vice Chair as necessary.
+  The Vice Chair has the authority of the Chair in the event of any absence or
+  unavailability of the Chair, and in such circumstances, any duty delegated
+  to the Chair herein MAY be performed by the Vice Chair. For example, the Vice
+  Chair MAY preside at SMCWG Meetings and Teleconferences in the Chair’s
+  absence.
+- SMCWG-created Subcommittees MAY be approved either:
+
+  1. by formal ballot as described in [6.1](#6.1-voting-structure), or
+  2. by simple majority vote of those members present at a regularly
+     scheduled SMCWG Meeting or Teleconference provided that the proposal is
+     mentioned in an agenda circulated on the SMCWG Mail List at least
+     forty-eight (48) hours prior to the SMCWG Meeting or Teleconference.
```

Comment:
This does not seem to have any precedent in our Forum procedures, and we've intentionally adopted procedures that prohibit such "decide on a call" shenanigans, especially because they're hostile to members in various timezones and jurisdictions.

For example, membership votes may be forced to ballot, but this doesn't have such a provision.

While subcommittees are much more lightweight than they were, and can reasonably tackle anything in scope, they can also be used as a means to stall work (by deferring to an unproductive subcommittee chartered to stall work).

So while removal of this second bullet is optional, it would be a serious blow to the legitimacy of the Forum if this clause were ever exercised and work deferred to such a subcommittee rather than take it on. So it might be better to just remove.